component/pages/pages.scala

```diff
@@ -144,13 +144,13 @@ case object RecommendMfaSkipAcknowledgePage extends FormPage {
   override val url: String = s"${Env.host}/developer/login/2SV-not-set"
 }
 
-case object Login2svEnterAccessCodePage extends FormPage {
+case object AuthAppLoginAccessCodePage extends FormPage {
   def clickContinue() = {
     click on id("submit")
   }
 
   override val pageHeading: String = "Enter your access code"
-  override val url: String = s"${Env.host}/developer/login-totp"
+  override val url: String = s"${Env.host}/developer/login-mfa?mfaId=13eae037-7b76-4bfd-8f77-feebd0611ebb&mfaType=AUTHENTICATOR_APP"
```

Comment:
I suggest this UUID and the other one at line 178 are put into constants so it's clearer when maintaining the tests. Alternatively, refer to the builder variables if possible: `authenticatorAppMfaDetails` and `smsMfaDetails`

Reply:
ok have pushed a change to try and address your comments